content/docs/projects/approver-policy.md

```diff
@@ -269,7 +269,7 @@ CertificateRequestPolicies against CertificateRequests for evaluation.  A
 CertificateRequestPolicy must select, and therefore match, a CertificateRequest
 for it to be considered for evaluation of the request.
 
-> ⚠️ Note that the user must still be bound by [RBAC](#Configuration) for
+> ⚠️ Note that the user must still be bound by [RBAC](#configuration) for
```

Comment:
The link checker picked this up before I disabled the `#anchor` checks.
It seems that the autogenerated markdown anchors are always lower case.